scirpt.js

```diff
@@ -1,5 +1,53 @@
 const humburger = document.querySelector('.humburger');
 const navMenu = document.querySelector('.new');
+const body = document.querySelector('body');
+const open = document.querySelector('.buttonButton1');
+open.addEventListener('click', () => {
+  const htm = document.createElement('div');
+  htm.innerHTML = `
```

Comment:
- [ ] Kindly style all the pop-up well
- Your Design
<img width="1309" alt="Screenshot 2023-05-18 at 04 23 47" src="https://github.com/abdozayan12/portfolio-include-js/assets/81444338/bd330dc4-a853-45ac-aa59-f5738d9d5fd6">

- Figma Design

<img width="1318" alt="Screenshot 2023-05-18 at 04 22 11" src="https://github.com/abdozayan12/portfolio-include-js/assets/81444338/ccc9c339-6291-45e8-83a6-5b4694a7ad77">